### ISSUE TITLE
Useless throws annotation not detected for children

### ISSUE THREAD

**Issue by janedbal** [pepakriz/phpstan-exception-rules#32]
Following code could be checked for useless throws annotation (`NotFoundException` extends `HttpException`).

```php
/**
 * @throws HttpException
 * @throws NotFoundException
 */
```

What do you think?

**Comment by marcospassos** [comment]
_Implementation note:_ it should check that `NotFoundException` is not a descriptive annotation.
